<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zhuce</class>
 <widget class="QDialog" name="Zhuce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎注册</string>
  </property>
  <widget class="QPushButton" name="signin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="zhuce_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>         欢迎注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="user_label">
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="user_line"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="key1_label">
      <property name="text">
       <string>输入密码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="key1_line"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="key2_lebel">
      <property name="text">
       <string>再次输入密码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="key2_line"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="tishikuang">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
